README.md

```diff
@@ -128,6 +129,18 @@ The image is automatically scaled to a 120 pixel square for the top of the Docke
 
 This file should contain a link to the maintainers of the Dockerfile.
 
+## `metadata.json`
+
+This file contains data about the repo for Docker Hub. The minimum file is defined below. `./metadata.sh [repo-name]` must be used to correctly format it (use `-f` to apply its suggested changes). There is a limit to the number of Docker Hub categories allowed; run with `-c` to check the limit and categories. `metadata.json` in the root contains the list of categories to choose from.
```

Comment:
Should we provide example usages?

```sh
$ ./metadata.sh -f -c REPO_NAME
```

Reply:
`-f` was changed to `-w` and now I dropped the `-c` and `-d` flags (commit incoming with script and README adjustments).